portfolio/src/main/webapp/script.js

```diff
@@ -13,9 +13,12 @@
 // limitations under the License.
 
 function getComments() {
-    fetch('/data').then(response => response.json()).then((comment) => {
+    var commentCount = document.getElementById('commentCount').value;
+    fetch('/data?count='+ commentCount).then(response => response.json()).then((comment) => {
```

Comment:
optional:

Consider using async-functions if you want.  Although be consistent within a file.  (Like either use them everywhere in the file or not at all.)

go/js-practices/async-functions 

async functions were introduced with ES 2017 and are the preferred way of interacting with Promises and other Thenables.

All async functions return a Promise which is resolved with the function's return value or rejected if there is any uncaught exception.

jankj@ has been using them, so you can ask him for any help on it.  (I actually haven't used any async functions myself, and I was just using promises.)

Reply:
From the tutorials they said that functionally async is the same and is just a preference. Is this incorrect?